connect_four_game.cpp

```diff
@@ -0,0 +1,203 @@
+
+/**
+* @file
+* @brief [Connect Four](https://en.wikipedia.org/wiki/Connect_Four)
+*
+* @details
+* This is a user interactive Connect-Four game for two players.
+* A board with 6 rows and 8 columns is displayed.
+* Two players are assigned the symbols "x" and "o".
+* They take turns to choose the column where their respective symbol can be inserted.
+* A player wins when 4 spaces are connected continuously: Horizontally, Vertically, Diagonally Forward, or Diagonally Backward.
+* The program declares the winner.
+*
+* @Connect Four trademark Milton Bradley, 1974
+* @author [Arunabh Bhattacharya](https://github.com/arunabh-cloud)
+*
+*/
+#include <iostream>
+using namespace std;
+using std::cout; // For displaying output
+using std::cin; // For taking user input
+
+// Global constants COLS and ROWS for the standard (row * column) 2D game board
+const int COLS = 8;
+const int ROWS = 6;
+
+// Function prototypes
+void printBoard(char [][COLS], int);
+void initializeBoard(char [][COLS], int);
+bool checkIfWinner(char [][COLS], int, int, char);
+
+/**
+ * @brief Main function
+ * @returns 0 on exit
+ */
+int main() {
```

Comment:
Hey there! Could you implement the main function into multiple functions, so that you can run them on the main function seamlessly and better? Let us know if you need any help. Thanks. 🙂

Reply:
Hi,
Made all the changes. 
Hopefully they will work.
Thanks.